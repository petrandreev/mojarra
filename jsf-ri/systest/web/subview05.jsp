<!--
 Copyright 2003 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->
<%@ page contentType="text/html" language="java" %>
<%@ taglib prefix="c" uri="http://java.sun.com/jstl/core" %>
<%@ taglib prefix="f" uri="http://java.sun.com/jsf/core" %>
<%@ taglib prefix="h" uri="http://java.sun.com/jsf/html" %>

<f:view>
<html>
<head>
<title>subview05</title>
</head>
<body>
<h:output_text value="Begin test jsp:include with subview and iterator tag in included page"/>
<br>
<jsp:include page="subviewIterator01.jsp"/>
<h:output_text value="Text from subview05.jsp"/>
<h:output_text value="End test jsp:include with subview and iterator tag in included page"/>
</body>
</html>
</f:view>
