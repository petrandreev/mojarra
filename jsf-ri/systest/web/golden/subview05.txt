<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->






<html>
<head>
<title>subview05</title>
</head>
<body>
<p>Begin test jsp:include with subview and iterator tag in included page</p>
<br />
<p><!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->







  <br>
  
    Array[0]: 
    This component has no ID <br>
    <input id="subviewInner:j_id_id17i0" type="text" name="subviewInner:j_id_id17i0" value="This component has no ID " /><br>
  
    Array[1]: 
    This component has no ID <br>
    <input id="subviewInner:j_id_id17i0j_id_1" type="text" name="subviewInner:j_id_id17i0j_id_1" value="This component has no ID " /><br>
  
    Array[2]: 
    This component has no ID <br>
    <input id="subviewInner:j_id_id17i0j_id_2" type="text" name="subviewInner:j_id_id17i0j_id_2" value="This component has no ID " /><br>
  
    Array[3]: 
    This component has no ID <br>
    <input id="subviewInner:j_id_id17i0j_id_3" type="text" name="subviewInner:j_id_id17i0j_id_3" value="This component has no ID " /><br>
  



</p>

<p>Text from subview05.jsp</p>
</p>End test jsp:include with subview and iterator tag in included page</p>
</body>
</html>

